### ISSUE TITLE
[Request] A few features I'd like to have

### ISSUE THREAD

**Issue by fuyushi** [neowutran/ShinraMeter#249]
Dont know how requests are handled but these are moreso suggestions i'd like to see:
1. Option to use windows TTS (Text-to-Speech) like ACT (Advanced combat tracker) does that's used as a "dps"-meter in other games, like ff14 or everquest instead of just the beeping
2. Option to display cooldowns of some other classes? Like adrenaline rush, or some debuffs. But that might be difficult with cdr glyphs and stats, so maybe just a timer. 
3. Uniform excel graphs, so that the times/graphs line up with each other and it's easier to read between them. With that a timeline of skills dealt would be useful.
4. A display for next natural enrage? They're usually 10% hp after the end of the last enrage, if not a display for the end of the last enrage(35-36s) would be good enough anyway

I want to say i'm not demanding any of these, I just thought they'd be useful/good ideas and wanted to share them so they might happen in the future

**Comment by neowutran** [comment]
A lot of good idea =) 
Didn't know TTS, will play a bit with that (btw, you can configure to play a sound file directly instead of beep)

**Comment by fuyushi** [comment]
Tried the idea to have a 34 seconds mp3 to indicate enrage is about to end, and now recently shinrameter freezes every now and then (no crash) and I also have the system process 'system interrupts' use up one whole cpu core and everything's choppy - especially video streams, audio and whatnot (windows 10). Immediately fixes itself when I close Tera. My friend uses the same thing on windows 7  and his audio regularly stops working, like steam, discord notifications and such and choppiness. Did I set it up wrong, or is this a bug? 

	
	<abnormality ingame="true" trigger="Added" target="Boss">
		<abnormalities>
			<abnormality>8888888</abnormality>
		</abnormalities>
		<actions>
			<notify>
				<balloon title_text="{abnormality_name} : on" body_text="{abnormality_name}" display_time="3000" />
<music file="E:\Programme (x86)\ShinraMeterV1.58\35senrage.mp3" volume="10" duration="34000"/>
			</notify>
		</actions>
	</abnormality>
